xla/service/gpu/tests/BUILD

```diff
@@ -718,6 +715,24 @@ lit_test_suite(
     ],
 )
 
+lit_test_suite(
```

Comment:
Could use prev target and just pass GPU explicitly? Don't need to use the GPU renaming?

Reply:
This comment, and enforce_glob, and specify the GPU in the file are all connected, so I'll just reply here.

In the BUILD file, when "if_cuda_is_configured" is true, we can only specify one value for the GPU param. So we can't have both ampere and hopper. So, if we keep a single BUILD rule to test both, then we can keep specifying ampere in the BUILD file, and in some files in the glob change xla_gpu_target_config_filename to be hopper (and those tests won't run on rocm either). I think this can lead to confusion when someone looking at the BUILD rule thinks all tests are ampere, but in reality some are hopper. 

If we split into two rules, one for ampere and one for hopper, then we can't use enforce_glob, because it enforces that all ".hlo" files are included in the BUILD rule, which will no longer be the case.